public/game-master/styles/game-master.styl

```diff
@@ -21,8 +21,15 @@
 		
 		.round-title
 			flex-grow 4
-		.round-timer
-			flex-grow 1
+			line-height 1.2em
```

Comment:
Not an expert here... Just googled several recommendations for font-size on a page - and most of them recommend `px` on html\body and `em` inside